saleor/graphql/product/bulk_mutations/products.py

```diff
@@ -799,13 +816,45 @@ class Meta:
         error_type_class = BulkStockError
         error_type_field = "bulk_stock_errors"
 
+    class Arguments:
+        variant_id = graphene.ID(
+            required=False,
+            description="ID of a product variant for which stocks will be created.",
+        )
+        sku = graphene.String(
+            required=False,
+            description="SKU of product variant for which stocks will be deleted.",
+        )
+        stocks = NonNullList(
+            StockInput,
+            required=True,
+            description="Input list of stocks to create.",
+        )
+
     @classmethod
     def perform_mutation(cls, _root, info, **data):
         errors = defaultdict(list)
         stocks = data["stocks"]
-        variant = cls.get_node_or_error(
-            info, data["variant_id"], only_type=ProductVariant
-        )
+        sku = data.get("sku")
+        variant_id = data.get("variant_id")
+
+        validate_one_of_args_is_in_mutation(
+            ProductErrorCode, "sku", sku, "variant_id", variant_id
+        )
+
+        if variant_id:
+            variant = cls.get_node_or_error(info, variant_id, only_type=ProductVariant)
+        else:
+            variant = models.ProductVariant.objects.filter(sku=sku).first()
```

Comment:
In the database, SKU could be "null". There is a change to perform this line with `sku=None`?

Reply:
`validate_one_of_args_is_in_mutation` will return error in that case